src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -3250,9 +3250,10 @@ paths:
                   items:
                     type: string
                   type: array
-                status:
-                  description: Status of the Host
-                  type: string
+                maintenance:
+                  default: false
+                  description: Enable Host Maintenance
+                  type: boolean
```

Comment:
This is a breaking change. Is this endpoint flagged as v0.x?

Reply:
Its flagged v0.1. But yeah I removed this change according to @avanthakkar 's suggestion.